.prow/config.yaml

```diff
@@ -225,6 +225,48 @@ presubmits:
     branches:
     - ^v0\.(3|4)-branch$
 
+    - name: test-end-to-end-batch-dataflow
+      decorate: true
+      always_run: true
```

Comment:
`always_run` should probably be off by default. Otherwise the service account can be printed out by any random person.